<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDateTimee</class>
 <widget class="QMainWindow" name="QtDateTimee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDateTimee</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>日期和时间</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>当前时间</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>字符串显示</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>时间</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTimeEdit" name="timeEdit">
         <property name="displayFormat">
          <string>HH:mm:ss</string>
         </property>
         <property name="time">
          <time>
           <hour>15</hour>
           <minute>3</minute>
           <second>1</second>
          </time>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="editTime">
         <property name="inputMask">
          <string>99:99:99;_</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnTime">
         <property name="text">
          <string>Time</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btnFormatTime">
         <property name="text">
          <string>设置时间</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>日期</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="dateEdit">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2022</year>
           <month>12</month>
           <day>31</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>yyyy-M-d</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="editDate">
         <property name="inputMask">
          <string>9999-99-99</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="btnDate">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="btnFormatDate">
         <property name="text">
          <string>设置日期</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>日期时间</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="date">
          <date>
           <year>2023</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>yyyy-M-d HH:mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="editDateTime">
         <property name="inputMask">
          <string>9999-99-99 99:99:99</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="btnDateTime">
         <property name="text">
          <string>DateTime</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="btnFormatDateTime">
         <property name="text">
          <string>设置日期时间</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>日历</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>日期</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editCalendar"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCalendarWidget" name="calendar"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
